src/main/resources/application.yml

```
@@ -9,6 +9,15 @@ spring:
     import:
       - optional:file:./.env[.properties]
       - classpath:config/oauth/security.yml
+  cloud:
+    openfeign:
+      client:
+        config:
+            default:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)